src/main/java/uk/gov/cslearning/record/service/xapi/XApiService.java

```diff
@@ -75,30 +78,23 @@ public XApiService(XApiProperties xApiProperties) {
         return statements;
     }
 
-    public Collection<Statement> getStatements(String activityId) throws IOException {
-        LOGGER.debug("Getting xAPI statements for activity {}", activityId);
-
-        StatementClient statementClient = new StatementClient(xApiProperties.getUrl(), xApiProperties.getUsername(),
-                xApiProperties.getPassword());
-
-        StatementResult result = statementClient
-                .filterByActivity(activityId)
-                .includeRelatedActivities(true)
-                .getStatements();
-
-        List<Statement> statements = new ArrayList<>(result.getStatements());
-        while (result.hasMore()) {
-            result = statementClient.getStatements(stripPath(xApiProperties.getUrl(), result.getMore()));
-            statements.addAll(result.getStatements());
-        }
-        return statements;
-    }
-
     private String stripPath(String url, String more) {
         int length = 1;
         while (!url.endsWith(more.substring(0, length))) {
             length += 1;
         }
         return more.substring(length);
     }
+
+    public void register(BookingDto bookingDto){
+
+        StatementClient statementClient = statementClientFactory.create();
```

Comment:
Possible Refactor - The contents of this method won't change for other operations. The only change will be `statement` body.  Private method to hold the body and public methods to explicitly create the statements maybe?
```
private postStatement(statement)
public register() {
  postStatement(createStatementForRegister)
}
```